### ISSUE TITLE
java.lang.NoSuchMethodError crash on Android 7.1

### ISSUE THREAD

**Issue by meierjan** [novoda/simple-chrome-custom-tabs#29]
```
java.lang.NoSuchMethodError: No static method startActivity(Landroid/app/Activity;Landroid/content/Intent;Landroid/os/Bundle;)V in class Landroid/support/v4/app/ActivityCompat; or its super classes (declaration of 'android.support.v4.app.ActivityCompat' appears in /data/user/0/net.nextbike.v3/files/.jrebel/load-bundles/libraries89_support-compat.zip)
at android.support.customtabs.CustomTabsIntent.launchUrl(CustomTabsIntent.java:200)
at com.novoda.simplechromecustomtabs.navigation.SimpleChromeCustomTabsWebNavigator.navigateTo(SimpleChromeCustomTabsWebNavigator.java:40)
at net.nextbike.v3.presentation.ui.main.view.MainActivity.showForMenuItem(MainActivity.java:305)
at net.nextbike.v3.presentation.ui.main.presenter.MainPresenter.onCustomMenuItemClicked(MainPresenter.java:121)
at net.nextbike.v3.presentation.ui.main.view.MainActivity.onNavigationItemSelected(MainActivity.java:135)
at android.support.design.widget.NavigationView$1.onMenuItemSelected(NavigationView.java:156)
at android.support.v7.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:817)
at android.support.v7.view.menu.SubMenuBuilder.dispatchMenuItemSelected(SubMenuBuilder.java:89)
at android.support.v7.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:156)
at android.support.v7.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:964)
at android.support.design.internal.NavigationMenuPresenter$1.onClick(NavigationMenuPresenter.java:332)
at android.view.View.performClick(View.java:5637)
at android.view.View$PerformClick.run(View.java:22429)
at android.os.Handler.handleCallback(Handler.java:751)
at android.os.Handler.dispatchMessage(Handler.java:95)
at android.os.Looper.loop(Looper.java:154)
at android.app.ActivityThread.main(ActivityThread.java:6119)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
```

The stack trace claims that there seems to be no method `ActivityCompat.startActivity(X)`. So when I look into the docs[1] it seems to be true.

1.  [`ActivityCompat` Documentation](https://developer.android.com/reference/android/support/v4/app/ActivityCompat.html)

**Comment by meierjan** [comment]
Oh, this is in `android.support.customtabs.CustomTabsIntent` so its not related to you guys

**Comment by juankysoriano** [comment]
@meierjan thank you for your contribution.

We have now released version `0.1.4` which is using the latest version of the support libraries, so you shouldn't see this issue again.

As an additional note we have not being able to reproduce this on any `Android 7.1` device (however we didn't had a Nexus 5X to test it.

**Comment by meierjan** [comment]
I think it's 7.1.1 on my nexus 5X
